### DISCUSSION TITLE
What's up cool pup / Any issues / Feature suggestions

### DISCUSSION THREAD

**Discussion by glenn2223** [glenn2223/vscode-live-sass-compiler#223] [General]
Hey @therealcoolpup,

I've moved this to a discussion to save spamming the [unrelated issue](https://github.com/glenn2223/vscode-live-sass-compiler/issues/222).

---

>`@glenn2223` All software, free or not is subject to criticism.

I love criticism - when it is criticism - it's how things get better. Tell me though, where is the criticism in `Pity there aren't many alternatives to this POS extension`? There is no mention of any issues or faults, it would be more helpful if you had specific feedback rather than a statement that doesn't help better the extension itself

>Why are you surprised you aren't making much from this? Thats the nature of open source, you give your everything and get nothing in return.

Yes, that is the disgusting state of open-source. Unfortunately I can't please everyone with the limited time I have on this, as it's not my full time job. So, that's why I'm one of the ones that's trying to change the dynamic. My open-source commitment states that the work I rely on gets a portion of any donations

---

>Digression. Im not sure who was controlling the vs code extension but the old one was working fine and now for some reason it is deprecated (installed new one and it works), why?

Digression... I love a good digression. Ritwick's extension hadn't had an update since 2018, this does not only have security risks but also made the extension slow and the code it was producing outdated too. I made so many changes to improve the speed an reliability of the extension. After a while Ritwick finally checked on this (really old) extension and saw that I had been answering his issues and actually had a decent fork. From this he decided to deprecate his extension in favour of my own (which I was glad to do by the way 😉)

---

Now, back from the digression. 

I will always do my best to fix issues or better the extension. What issue are you having with the extension? Or, what are your feature suggestions?

**Comment by KohlerVG** [comment]
Thank you for your hard work on this extension Glenn!

**I have a small feature request that I think would be great**
The Live SASS Compiler errors only appear in the terminal, it would be nice have the errors show in a dialog box instead, i think there is a VSCode dialog function you can use to do this...

**Reply by glenn2223** [reply]
Thanks for opening your discussion; this particular discussion was to get to the bottom of coolpups issues with the extension. He's not offer any more info/details - as they often don't - so I'm going to close this discussion to save any future comments/confusion.

---

To those coming in the future, always open your own discussion unless you can resolve/help the OP. Things like _"Me too"_ (not this one 😜) aren't helpful on issues or discussions
